src/server/services/wmo.test.ts

```diff
@@ -1,10 +1,301 @@
+    expect(transformed.steps[0].isChecked).toBe(true);
+
+    expect(transformed.steps[1].isActive).toBe(true);
+    expect(transformed.steps[1].isChecked).toBe(true);
+
+    expect(transformed.steps[2].isActive).toBe(false);
+    expect(transformed.steps[2].isChecked).toBe(false);
+
+    expect(transformed.steps[3].isActive).toBe(false);
+    expect(transformed.steps[3].isChecked).toBe(false);
+  });
+
+  test('hulp bij het huishouden (tijdelijk gestopt)', () => {
+    const [transformed] = transformWmoResponse([apiTestItems[2]], compareDate);
+    expect(transformed.steps.length).toBe(4);
+    expect(transformed.isActual).toBe(true);
+    expect(transformed.steps[0].isActive).toBe(false);
+    expect(transformed.steps[0].isChecked).toBe(true);
+
+    expect(transformed.steps[1].isActive).toBe(false);
+    expect(transformed.steps[1].isChecked).toBe(true);
+
+    expect(transformed.steps[2].isActive).toBe(true);
+    expect(transformed.steps[2].isChecked).toBe(true);
+
+    expect(transformed.steps[3].isActive).toBe(false);
+    expect(transformed.steps[3].isChecked).toBe(false);
+  });
+
+  test('hulp bij het huishouden (gestopt)', () => {
+    const [transformed] = transformWmoResponse([apiTestItems[3]], compareDate);
+    expect(transformed.steps.length).toBe(4);
+    expect(transformed.isActual).toBe(false);
+    expect(transformed.steps[0].isActive).toBe(false);
+    expect(transformed.steps[0].isChecked).toBe(true);
+
+    expect(transformed.steps[1].isActive).toBe(false);
+    expect(transformed.steps[1].isChecked).toBe(true);
+
+    expect(transformed.steps[2].isActive).toBe(false);
+    expect(transformed.steps[2].isChecked).toBe(true);
+
+    expect(transformed.steps[3].isActive).toBe(true);
+    expect(transformed.steps[3].isChecked).toBe(true);
+  });
+
+  test('financiële tegemoetkoming gebruik gesloten buitenwagen in bruikleen', () => {
+    const [transformed] = transformWmoResponse([apiTestItems[4]], compareDate);
+    expect(transformed.steps.length).toBe(2);
+    expect(transformed.isActual).toBe(true);
+    expect(transformed.steps[0].isActive).toBe(true);
+    expect(transformed.steps[0].isChecked).toBe(true);
+
+    expect(transformed.steps[1].isActive).toBe(false);
+    expect(transformed.steps[1].isChecked).toBe(false);
+  });
+
+  test('PGB item 1', () => {
+    const testItem = apiTestItems[5];
+    let [transformed] = transformWmoResponse([testItem], compareDate);
+    expect(transformed.steps.length).toBe(2);
+    expect(transformed.isActual).toBe(false);
+
+    expect(transformed.steps[0].isActive).toBe(false);
+    expect(transformed.steps[0].isChecked).toBe(true);
+
+    expect(transformed.steps[1].isActive).toBe(true);
+    expect(transformed.steps[1].isChecked).toBe(false);
+  });
+
+  test('All transformations', () => {
+    let transformed = transformWmoResponse(apiTestItems, compareDate);
+    expect(transformed).toMatchSnapshot();
+  });
+
+  test('hasFutureDate', () => {
```

Comment:
Denk dat het goed is om bij deze test en bij `hasHistoricDate` ook nog de test case toe te voegen met twee keer `compareDate` als parameter.

Reply:
Ja, goed plan.